qemu/tests/block_copy.py

```diff
@@ -392,3 +392,28 @@ def clean(self):
         while self.trash_files:
             tmp_file = self.trash_files.pop()
             utils.system("rm -f %s" % tmp_file, ignore_status=True)
+
+    def create_file(self, file_name):
+        """
+        Create file and record m5 value of them.
+        :param file_name: the file need to be created
+        """
+        params = self.params
+        session = self.get_session()
+        file_create_cmd = params.get("create_cmd", "touch %s")
+        test_exists_cmd = params.get("test_exists_cmd", "test -f %s")
+        if session.cmd_status(test_exists_cmd % file_name):
+            session.cmd(file_create_cmd % file_name)
+        session.cmd("md5sum %s > %s.md5" % (file_name, file_name))
+        session.cmd("sync & echo 3 > /proc/sys/vm/drop_caches")
```

Comment:
Not here I think, for creating file inside guest, only need to flush data inside guest. It is needed when block job completed so still keep calling  utils_memory.drop_caches() after backup, with line 113 in live_backup_base.py